testsuite/python/tabulated_sympy.py

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2013-2022 The ESPResSo project
```

Comment:
The license header should reflect the year range you actually worked on the code, plus the years of the original code if you copied parts of it from somewhere else.